src/main/java/org/gridsuite/loadflow/server/entities/LoadFlowResultEntity.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.loadflow.server.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.ZonedDateTime;
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * @author Anis Touri <anis.touri at rte-france.com>
+ */
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name = "loadflow_result")
+public class LoadFlowResultEntity {
+
+    @Id
+    private UUID resultUuid;
+
+    @Column
+    private ZonedDateTime writeTimeStamp;
+
+    @OneToMany(cascade = CascadeType.ALL)
```

Comment:
Why using a Set and a oneToMany relation ? 
We mostly use : 
@ElementCollection
@CollectionTable(name = "sensitivity")
 private List<SensitivityEmbeddable> sensitivities;

And it was implemented this way in the study-server DB

Reply:
Without a @JoinColumn and without a mapped @ManyToOne on the other side, Hibernate assumes that you want to use a join table strategy. You are correct, I have added the @JoinColumn.